corehq/apps/hqwebapp/views.py

```diff
@@ -579,7 +579,26 @@ def jserror(request):
 @method_decorator([login_required], name='dispatch')
 class BugReportView(View):
     def post(self, req, *args, **kwargs):
-        report = dict([(key, req.POST.get(key, '')) for key in (
+        email = self._get_email_message(
+            post_params=req.POST,
+            couch_user=req.couch_user,
+            uploaded_file=req.FILES.get('report_issue')
+        )
+
+        email.send(fail_silently=False)
+
+        if req.POST.get('five-hundred-report'):
+            messages.success(
+                req,
+                "Your CommCare HQ Issue Report has been sent. We are working quickly to resolve this problem."
```

Comment:
it'd be nice to add translation

Reply:
Okay, pulled that out into it's own PR: https://github.com/dimagi/commcare-hq/pull/27130